components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -157,8 +241,29 @@ subroutine crm_physics_register()
    call pbuf_add_field('CRM_CLD_RAD', 'physpkg', dtype_r8, dims_crm_rad, idx)
    call pbuf_add_field('CRM_QRAD',    'global',  dtype_r8, dims_crm_rad, idx)
    
-   cldo_idx = pbuf_get_index('CLDO')
-   call pbuf_add_field('CLDO', 'global', dtype_r8, (/pcols ,pver, dyn_time_lvls/), cldo_idx  )
+   call pbuf_add_field('CLDO',  'global', dtype_r8, (/pcols, pver, dyn_time_lvls/), cldo_idx  )
+   call pbuf_add_field('CLD',   'global', dtype_r8, (/pcols, pver, dyn_time_lvls/), cld_idx)
+   call pbuf_add_field('CONCLD','global', dtype_r8, (/pcols, pver, dyn_time_lvls/), concld_idx)
+   
+   call pbuf_add_field('QME',   'physpkg',dtype_r8, (/pcols, pver/), qme_idx)
+   call pbuf_add_field('PRAIN', 'physpkg',dtype_r8, (/pcols, pver/), prain_idx)
+   call pbuf_add_field('NEVAPR',     'physpkg',dtype_r8,(/pcols,pver/), nevapr_idx)
+   call pbuf_add_field('PRER_EVAP',  'global', dtype_r8,(/pcols,pver/), prer_evap_idx)
+
+   call pbuf_add_field('WSEDL',      'physpkg',dtype_r8,(/pcols,pver/), wsedl_idx)
+
+   call pbuf_add_field('REI',     'physpkg',dtype_r8,(/pcols,pver/), rei_idx)
+   call pbuf_add_field('REL',     'physpkg',dtype_r8,(/pcols,pver/), rel_idx)
+   call pbuf_add_field('DEI',     'physpkg',dtype_r8,(/pcols,pver/), dei_idx)        ! Mitchell ice effective diameter for radiation
+   call pbuf_add_field('MU',      'physpkg',dtype_r8,(/pcols,pver/), mu_idx)         ! Size distribution shape parameter for radiation
```

Comment:
These don't exist without MG2, do they? If we aren't using MG2, then I think we need to use the "old" cloud optics params that depend on rel and rei, not on mu and lambdac, and MU and LAMBDAC have no place in the pbuf if they are undefined.

Reply:
There's a few variables like this that are used in places like cam_diagnostics and elsewhere. The "CUSH" variable is like this. It's definitely not going to be used for the MMF, but I would have to modify these other files to add the "if use_MMF" logic to clean it up. At this point, none of these are hurting anything and it should be BFB to do more clean up like this later on, so I saw we punt on figuring out stuff like this unless it's obvious we can just delete a few lines.
Plus, I think we will need these variables when we switch to P3.

Reply:
Fair enough.